src/topology/topological_fiber_bundle.lean

```diff
@@ -391,7 +391,7 @@ end
 that all the local trivialization are continuous. -/
 instance to_topological_space : topological_space Z.total_space :=
 topological_space.generate_from $ ⋃ (i : ι) (s : set (B × F)) (s_open : is_open s),
-  {(Z.local_triv' i).source ∩ (Z.local_triv' i) ⁻¹' s}
+  singleton $ (Z.local_triv' i).source ∩ (Z.local_triv' i) ⁻¹' s
```

Comment:
Is this a regression?

Reply:
It's an unwanted side effect of @Vierkantor's set replacement notation.  `{(` now has a special meaning.

Reply:
Yes, this seems to be unintended fallout from https://github.com/leanprover-community/lean/pull/402

Reply:
I can keep fixing these but it's more and more annoying. There's lots of `{(1 : A)}` etc in mathlib which seems totally acceptable.

Reply:
Should we do 3.18.2?  leanprover-community/lean#415

Reply:
I think we can revert this one as well now.

Reply:
I'm pretty sure I tried it and it still fails. Let me see again.

Reply:
```
/home/rob/lean/mathlib/src/topology/topological_fiber_bundle.lean:394:20: error: invalid set replacement notation, ',', '}', or `|` expected
/home/rob/lean/mathlib/src/topology/topological_fiber_bundle.lean:394:20: error: invalid field notation, type is not of the form (C ...) where C is a constant
  {Z.local_triv' i}
has type
  ?m_1
/home/rob/lean/mathlib/src/topology/topological_fiber_bundle.lean:394:48: error: type mismatch at application
  preimage ⇑(Z.local_triv' i)
term
  ⇑(Z.local_triv' i)
has type
  has_coe_to_fun.F (Z.local_triv' i) : Type (max u_2 u_3)
but is expected to have type
  set Z.total_space → ?m_1 : Type (max (max u_2 u_3) ?)
/home/rob/lean/mathlib/src/topology/topological_fiber_bundle.lean:394:53: error: command expected
```

Reply:
It works with extra parens: `{((Z.local_triv' i).source ∩ (Z.local_triv' i) ⁻¹' s)}`

https://github.com/leanprover-community/lean/issues/423

Reply:
I was able to revert it with 3.18.4.